Dockerfile_Fedora

```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
 
 
 LABEL project="gluster-build"
-LABEL maintainer "sajmoham@redhat.com"
+LABEL maintainer "gluster-users@gluster.org"
 
-RUN dnf -y update && dnf clean all
+RUN dnf -y --setopt tsflags=nodocs install automake autoconf libtool flex bison openssl-devel  \
+    libxml2-devel python-devel libaio-devel librdmacm-devel readline-devel \
```

Comment:
Are you sure we need librdmacm-devel ? I build on Fedora without it.

Reply:
yes not required, verified it...